canonical-web-and-design/microk8s.io
# Add lazysizes.js

## Done

- Added lazysizes.js
- Set iframe on homepage to lazyload

## QA

- Visit http://localhost:8027/
- Open network panel in dev tools and clear the log
- Scroll down the page until you see the media player
- See that there is a new request in the network panel to asciinema.org
